cgo/pow/pow.go

```diff
@@ -11,6 +11,10 @@ extern bool dcurl_init();
 extern bool dcurl_destroy();
 extern int8_t *dcurl_entry(int8_t *trytes, int mwm, int threads);
 
+bool dcurl_init_wrapper(){
+	return dcurl_init(NULL);
```

Comment:
Does it really have to use `NULL`?

Reply:
It can be used to pass a configuration file in dcurl.

Reply:
Sorry I forgot that `dcurl_init()` do need a pointer as its parameter.
I think `extern bool dcurl_init();` might need a little change?

Reply:
I resolve it to `dcurl_init(void* config)`.

Reply:
Not really sure how to use the config file.